kairos-deposit-contract/malicious-session/src/main.rs

```diff
@@ -0,0 +1,15 @@
+#![no_std]
+#![no_main]
+use casper_contract::contract_api::{account, runtime, system};
+use casper_types::{runtime_args, AddressableEntityHash, URef, U512};
+
```

Comment:
Can we add a comment here on what a `malicious-session` does/ is supposed to do?

Reply:
I will add a comment explaining the session codes on top of each file.